index.md

```diff
@@ -84,9 +111,17 @@ This pipeline will produce a number of directories and files:
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
 results.
 
-## About
-This pipeline is part of [BioWDL](https://biowdl.github.io/)
-developed by [the SASC team](http://sasc.lumc.nl/).
+## Scattering
+The pipeline performs scattering to speed up analysis on grid computing
+clusters. This means that the reference genome will be split into regions of
+roughly equal size (see the `scatterSize` inputs). For certain steps each of
+these regions will be analyzed in separate jobs, allowing them to be processed
+in parallel.
+
+## Tool versions
+Included in the repository is an `environment.yml` file. This file includes
+all the tool version on which the workflow was tested. You can use conda and
+this file to create an environment with all the correct tools.
 
 ## Contact
 <p>
```

Comment:
Any question about running this pipeline and feature request such as adding additional tools and options, please contact sasc@lumc.nl